backend/config.default.yaml

```diff
@@ -91,6 +91,9 @@ migration:
   POPULATE_VERSION_MIGRATION:
     enable: true
     record_update_limit: 100
+  COLLABORATOR_RESOURCE_MIGRATION:
+    enable: true
```

Comment:
This should be done automatically on every bringup. Then when we want to disable, we should delete the code. The days of manual migration need to be gone.

Reply:
Actually, this doesn't even seem to be used. So that's fine, but it's confusing.

Reply:
This is used here: https://github.com/VertaAI/modeldb/blob/aj/feature/VR-8152_backfill_existing_project_resources/backend/src/main/java/ai/verta/modeldb/utils/ModelDBHibernateUtil.java#L762

Reply:
k